raisinbread/instruments/EEG_Acquisition_form.linst

```diff
@@ -0,0 +1,95 @@
+title{@}EEG_Acquisition_form
+header{@}{@}EEG Acquisition form
```

Comment:
```suggestion
header{@}{@}EEG Acquisition Form
```
capital F

Reply:
Can this line be removed or Is this needed?  
See your screenshots - there's an extra "EEG Acquisition" header line in the form above General Informations which should be removed.